auth/client/src/main/resources/schema/encrypted-expression-1_0.xsd

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <!--
+      ~ JBoss, Home of Professional Open Source.
+      ~ Copyright 2023 Red Hat, Inc., and individual contributors
+      ~ as indicated by the @author tags.
+      ~
+      ~ Licensed under the Apache License, Version 2.0 (the "License");
+      ~ you may not use this file except in compliance with the License.
+      ~ You may obtain a copy of the License at
+      ~
+      ~     http://www.apache.org/licenses/LICENSE-2.0
+      ~
+      ~ Unless required by applicable law or agreed to in writing, software
+      ~ distributed under the License is distributed on an "AS IS" BASIS,
+      ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      ~ See the License for the specific language governing permissions and
+      ~ limitations under the License.
+    -->
+
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xss="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="urn:encrypted:expression:1.0"
+            xmlns="urn:encrypted:expression:1.0"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="1.0">
+
+    <!-- Encrypted Expression elements -->
+    <xsd:element name="encrypted-expression" type="encrypted-expression-type"/>
+
+    <xsd:complexType name="encrypted-expression-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                Configuration for encrypted expressions and expression-resolvers to be used across different client configurations.
```

Comment:
Just a total minor, I would rename the file to encrypted-expressions-1_0.xsd

Reply:
fixed!

Reply:
@Skyllarr @PrarthonaPaul To be consistent with the other clients, just wondering if this should be encrypted-expressions-client-1_0.xsd or encryption-client-1_0.xsd with the root element updated accordingly as well.

Reply:
@fjuma Yes that is a good idea!

Reply:
sure! I can make changes to the schema and the elements.